README.md

```diff
@@ -87,6 +87,23 @@ Viewing word clouds doesn't require login access. The URL of the view page is in
         font size, the higher the voice pitch.
         - **"Apply conventional weighting"**: When enabled, more frequently submitted words will be highlighted.
 
+### Simulate a Word Cloud
+
+You can find a Node.js script to generate or update existing word cloud answers.
```

Comment:
Since there is only one script involved, it might be easier to read by adding the script name and a link of it at the beginning. For example: change "a Node.js script" to "a Node.js script [src/scripts/updateAnswers.js](./src/scripts/updateAnswers.js)".